docs/book/src/getting-started/testdata/project/internal/controller/memcached_controller.go

```diff
@@ -321,42 +220,8 @@ func (r *MemcachedReconciler) deploymentForMemcached(
 		},
 		Spec: appsv1.DeploymentSpec{
 			Replicas: &replicas,
-			Selector: &metav1.LabelSelector{
-				MatchLabels: ls,
-			},
 			Template: corev1.PodTemplateSpec{
-				ObjectMeta: metav1.ObjectMeta{
-					Labels: ls,
-				},
```

Comment:
TODO(camila): we will need to add 1 label due the test-controller implementation